kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/KroxyliciousService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kroxylicious;
+
+import java.util.Objects;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.Environment;
+import io.kroxylicious.systemtests.resources.manager.ResourceManager;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyConfigTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyDeploymentTemplates;
+import io.kroxylicious.systemtests.templates.kroxylicious.KroxyServiceTemplates;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Kroxy.
+ */
+public class KroxyliciousService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(KroxyliciousService.class);
+    private final String deploymentNamespace;
+    private final String containerImage;
+    private final ResourceManager resourceManager = ResourceManager.getInstance();
+
+    /**
+     * Instantiates a new KroxyliciousService to be used in kubernetes.
+     *
+     * @param deploymentNamespace the deployment namespace
+     */
+    public KroxyliciousService(String deploymentNamespace) {
+        this.deploymentNamespace = deploymentNamespace;
+        String kroxyUrl = Environment.KROXY_IMAGE_REPO + (Environment.KROXY_IMAGE_REPO.endsWith(":") ? "" : ":");
+        if (!Objects.equals(Environment.QUAY_ORG, Environment.QUAY_ORG_DEFAULT)) {
```

Comment:
by defeault `kroxyUrl` takes the value from the environment variable `KROXY_IMAGE_REPO` but for testing purposes I left the `QUAY_ORG` variable just in case we want to launch it locally with our current changes not merged to main. So setting `QUAY_ORG=fvila` for example will look for kroxy image in my quay repository, but as I said setting the corresponding value to the `KROXY_IMAGE_REPO` variable can take any other container registry.

If you agreed to remove this code, I can do it, it is just a lazy way to set the repository.

Reply:
I'm relaxed about this now. We can refactor later to kill the references. My point is that we should get out of the habit assuming everything is on quay.io.  In practice it is as simple as accepting a full registry url.

Reply:
No problem with removing that. I was the first approach and maintained for human lazy mode when running the tests locally.